app/models/manageiq/providers/azure/cloud_manager/provision/cloning.rb

```diff
@@ -1,19 +1,14 @@
 module ManageIQ::Providers::Azure::CloudManager::Provision::Cloning
-  def do_clone_task_check(clone_task_ref)
+  def do_clone_task_check(_clone_task_ref)
     source.with_provider_connection do |azure|
       vms      = ::Azure::Armrest::VirtualMachineService.new(azure)
-      instance = vms.get(clone_task_ref[:vm_name], clone_task_ref[:vm_resource_group])
```

Comment:
This was the only consumer of the values in the hash that was returned by `start_clone` and the values it needs are easily gotten from the Provision instance anyway.